entries/jQuery.htmlPrefilter.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<entry type="method" name="jQuery.htmlPrefilter" return="String">
+  <title>jQuery.htmlPrefilter()</title>
+  <desc>Modify and filter HTML strings passed through <a href="/category/manipulation/">jQuery manipulation methods</a>.</desc>
+  <signature>
+    <added>1.12/2.2</added>
+    <argument name="html" type="String">
+      <desc>The html string on which to operate.</desc>
+    </argument>
+  </signature>
+  <longdesc>
+    <p>This method rarely needs to be called directly. Instead, use it as an entry point to modify existing <a href="/category/manipulation/">jQuery manipulation methods</a>. For instance, to remove all scripts from incoming html strings, do this:</p>
+
+    <pre><code>
+$.htmlPrefilter = function( html ) {
+  return html.replace( /&lt;script&#91;^&lt;&#93;*&lt;\/script&gt;/g, "" );
+};
+    </code></pre>
+
+    <p>This function can also be overwritten in order to bypass certain edge case issues. The default `htmlPrefilter` function in jQuery will greedily ensure that all tags are XHTML-compliant. This includes anything that looks like an HTML tag, but is actually within a string (e.g. <pre>&lt;a title="&lt;div /&gt;"&gt;&lt;&gt;</pre>). The `jQuery.htmlPrefilter()` function can be used to bypass this:</p>
```

Comment:
This paragraph contains markdown (specifically backticks). It should use the `code` tag instead.